.github/shared-actions/update-status/action.yaml

```diff
@@ -0,0 +1,52 @@
+name: Build and Tests
+description: Build and tests
+inputs:
+  token:
+    description: "Github token"
+    required: true
+  dispatch_id:
+    description: 'Dispatch Id (repo/sha)'
+    required: true
+  state:
+    description: "Check State"
+    default: "pending"
+  description:
+    description: "Check description"
+    required: true
+  owner:
+    description: "Repo Owner"
+    default: "frontegg"
+    required: false
+  run_context:
+    description: "Run Context"
+    default: "frontegg/e2e-system-tests"
+    required: false
+
+runs:
+  using: "composite"
+  steps:
+    - name: Update PR Status
+      uses: actions/github-script@v5
+      env:
+        dispatch_id: ${{ inputs.dispatch_id }}
+        state: ${{ inputs.state }}
+        description: ${{ inputs.description }}
+        owner: ${{ inputs.owner }}
+        run_context: ${{ inputs.run_context }}
+      with:
+        github-token: ${{ inputs.token }}
+        script: |
+          const {dispatch_id,state, description, owner, run_context} = process.env
```

Comment:
you know you can use github.input :)

Reply:
<img width="961" alt="Screenshot 2024-01-11 at 16 55 39" src="https://github.com/frontegg/workflows/assets/69419539/cdbdbb46-a3d7-4e13-9b21-f3ef5bbed41b">